tests/nnstreamer_filter_extensions_common/unittest_tizen_template.cc

```diff
@@ -113,7 +113,7 @@ TEST (nnstreamer_EXT_NICK_NAME_basic_functions, open_close_01_n)
   GstTensorFilterProperties prop = {
     .fwname = "EXT_NAME",
     .fw_opened = 0,
-    .model_files = const_cast<const char **>(model_files),
+    .model_files = const_cast < const char **>(model_files),
```

Comment:
I am confusing now. 
  1) const_cast<const char **> 
  2) const_cast< const char ** > 
  3) const_cast< const char **>                  <--- yours

Which one is right?

Reply:
I ran the command from the [script](https://github.com/GStreamer/common/blob/master/gst-indent)  which gave me this result. Not sure which one is right.

Reply:
Actually, I'm not sure about `gst-indent` whether it's working well for C++.
Some of its recommendation are somewhat weird.

Reply:
@wooksong out of the 3 options, which indentation style do you recommend?

Reply:
IMO, first one is better :) <const char **>

Reply:
```gst-indent``` is for C code. Don't apply it to C++ codes.